### PR TITLE
FOUR-9010: Define the order to show the fields

### PR DESCRIPTION
**Pull request by CarliPinell**
## Issue & Reproduction Steps
Define the order to show the fields

## Related Tickets & Packages
- FOUR-9010

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

## Environment
ci:deploy
ci:package-data-sources:feature/FOUR-8121
ci:package-translations:feature/FOUR-8121
ci:package-vocabularies:feature/FOUR-8121
ci:connector-send-email:feature/FOUR-8121
ci:package-webentry:feature/FOUR-8121
ci:package-decision-engine:feature/FOUR-8121
ci:package-dynamic-ui:feature/FOUR-8121
ci:package-savedsearch:feature/FOUR-8121
ci:package-pm-blocks:feature/FOUR-8121
ci:package-collections:feature/FOUR-8121
ci:package-files:feature/FOUR-8121

### CODE REVIEW

File: resources/js/admin/users/components/SecurityLogsModal.vue [hunk 1]
```diff
@@ -191,7 +192,38 @@ export default {
           auxArray[this.capitalizeKey(key)] = this.booleanToString(value);
         }
       }
-      return auxArray;
+
+      auxArraySort  = this.sortModalArray(auxArray);
+
+      return auxArraySort;
```
Comment (fagubla) [inline resources/js/admin/users/components/SecurityLogsModal.vue:198]: change to `return this.sortModalArray(ausArray)`

File: resources/js/admin/users/components/SecurityLogsModal.vue [hunk 2]
```diff
@@ -191,7 +192,38 @@ export default {
...
+    },
+    /**
+     * Sort modal Array
+     */
+    sortModalArray(auxArray) {
+      let sortKey = ["Name"];
+      let auxArraySorted = {};
+      let specialKey = Object.keys(auxArray).find(key => ["Created_at", "Deleted_at", "Updated_at", "Last_modified", "Accessed_at"].includes(key));
```
Comment (fagubla) [inline resources/js/admin/users/components/SecurityLogsModal.vue:206]: this is a const never changes,
change the name to dateKey

**Review by fagubla — CHANGES_REQUESTED** [review]

File: resources/js/admin/users/components/SecurityLogsModal.vue (hunk 1 above)
Comment (CarliPinell) [reply resources/js/admin/users/components/SecurityLogsModal.vue:198]: Solved.

File: resources/js/admin/users/components/SecurityLogsModal.vue (hunk 2 above)
Comment (CarliPinell) [reply resources/js/admin/users/components/SecurityLogsModal.vue:206]: Solved.

**Review by fagubla — APPROVED** [review]

**Review by pmPaulis — APPROVED** [review]

**Pull request merged**